ctms/app.py

```diff
@@ -1122,5 +1124,41 @@ def stripe_pubsub(
     return {"status": "OK", "count": count}
 
 
+@app.delete(
+    "/ctms/gdpr/{email}",
+    tags=["Public"],
+    summary="delete email from database",
```

Comment:
Something like `Delete all contact information from primary email`?

Reply:
updated the summary :+1: